### PR TITLE
Makes the codebase PEP8 compliant

### LINKED ISSUE #17: Make the codebase PEP8 compliant
**Actual Behaviour** 
* The code is not organised and doesn't follow PEP8 standards

**Expected Behaviour**
* Make the codebase PEP8 compliant 
* Name the variables with meaningful names

### PR DESCRIPTION
**Pull request by agarwalrounak**
Fixes #17 

Changes: 
- The codebase is PEP8 compliant
- The variables have meaningful names

### CODE REVIEW

File: imdb.py [hunk 1]
```diff
@@ -8,140 +8,141 @@
...
-status=open('info.txt','a')
-
+status=open('info.txt', 'a')
 
 
 def info_movie():
     name = raw_input('\nEnter the title of the movie: ')
-    t = name.replace(' ','%20')
-    url = 'https://api.themoviedb.org/3/search/movie?api_key=ffb07b773769d55c36ccd83845385205&language=en-US&query='+str(t)+'&page=1&include_adult=false'
+    web_name = name.replace(' ', '%20')
```
Comment (akshatnitd) [inline imdb.py]: Make it `movie_name`

File: imdb.py [hunk 2]
```diff
@@ -8,140 +8,141 @@
...
+    url = 'https://api.themoviedb.org/3/search/movie?api_key=ffb07b773769d55c36ccd83845385205&language=en-US&query=' + str(web_name) + '&page=1&include_adult=false'
```
Comment (akshatnitd) [inline imdb.py]: Write meaningful names for `url` and `url2`

File: imdb.py [hunk 3]
```diff
@@ -8,140 +8,141 @@
...
     response = requests.get(url)
-    u = json.loads(response.text)
-    results  = u['results']
+    results_json = json.loads(response.text)
+    results  = results_json['results']
     id = results[0]['id']
```
Comment (akshatnitd) [inline imdb.py]: `movie_id`

File: imdb.py [hunk 4]
```diff
@@ -8,140 +8,141 @@
...
-		status.write ("\n\t PLOT        : \t\t"+plot)
+        title = details['title']
+        imdb_id = details['imdb_id']
+        year = details['release_date']
+        genre = details['genres']
+        language = details['spoken_languages']
+        duration = details['runtime']
+        plot = details['overview']
+
+        url3 = 'http://www.imdb.com/title/' + str(imdb_id)
```
Comment (akshatnitd) [inline imdb.py]: Rename it

**Comment by agarwalrounak** [comment]
@akshatnitd This PR got accidentally closed while rebasing. I just can't understand how. I will send a new PR soon.

**Pull request closed without merging**